src/main/java/com/esri/samples/scene/open_mobile_scene_package/README.md

```diff
@@ -0,0 +1,35 @@
+# Open mobile scene package
+
+Open and display a scene from an offline Mobile Scene Package (.mspk).
+
+![](OpenMobileScenePackageSample.png)
+
+## Use Case
+A .mspk file is an archive containing the data (specifically, basemaps and features) used to display an offline 3D scene.
+
+## How it works
+1. Create a `String` pointing to the samples-data/mspk directory which hosts the .mspk data.
+1. Use the static method `MobileScenePackage.isDirectReadSupportedAsync(mspkData)` to check whether the package can be read in the archived form (.mspk) or whether it needs to be unpacked.
+2. If the mobile scene package requires unpacking, use `MobileScenePackage.unpackAsync(mspkPath, pathToUnpackTo)` and wait for this to complete.
+3. Instantiate two `MobileScenePackage` objects, one using the path to the local `.mspk` file and the other to the unpacked directory.
+4. Call `mobileScenePackage.loadAsync` and check for any errors.
+5. When the mobile scene package is loaded, obtain the first `Scene` in the package using `mobileScenePackage.getScenes().get(0))`.
+6. Set the `SceneView` scene to the one obtained in the previous step.
```

Comment:
In general, don't refer to previous steps. Maybe something like this:

5. Load the mobile scene package. When finished, get the `ArcGISScene` objects inside with `mobileScenePackage.getScenes()`.
6. Set the first scene in the collection on the scene view with `sceneView.setArcGISScene(scene)`.